hushline/__init__.py

```diff
@@ -77,6 +77,14 @@ def inject_user() -> dict[str, Any]:
             return {"user": user}
         return {}
 
+    @app.context_processor
+    def inject_host() -> dict[str, HostOrganization]:
```

Comment:
If the reason for doing an insert here is to ensure the app "just works" out of the box, maybe it makes more sense to do a `.one_or_none()` query and then return something like `HostOrganization.default()` that's maintained in pure python.

I'm still really iffy on this pattern as it feels like we're going to be populating the DB in too many places, and IMO the only places it should happen are:

- migrations (for prod)
- a CLI (for prod/dev)
- `dev_data.py` for dev

If we start scattering this sort of stuff throughout the app, I'm not sure about long term maintainability.

Reply:
Thinking about this again that maybe a happy solution would be to inject a default value to the context but to _not_ add it to `db.session` and commit it. We should maybe have a global var and warn once (or maybe do this in `app.before_first_request` or something just so admins can see that maybe something is fucky if they don't have a row in the DB.